<launch>

  <group ns="franka">
    
    <arg name="robot_ip" default="172.16.0.2"/>
    <arg name="load_gripper" default="false"/>
    <include file="$(find franka_control)/launch/franka_control.launch" >
      <arg name="robot_ip" value="$(arg robot_ip)" />
      <arg name="load_gripper" value="$(arg load_gripper)" />
    </include>

    <rosparam command="load" file="$(find panda_controllers)/config/panda_controllers.yaml" />
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="cartesian_pose_controller_cino"/>

    <node pkg="hl_planning" type="straight_line" name ="tactip_node_1" output="screen">
      <param name = "contact_force"     type = "double" value = "0.5" />
      <param name = "ee_displacement_x" type = "double" value = "0.0" />
      <param name = "ee_displacement_y" type = "double" value = "0.25" />
      <param name = "duration"          type = "double" value = "10.0" />
    </node>

  </group>

</launch>